Frameworks/AddressBook/ABMultiValueInternal.mm

```diff
@@ -159,4 +159,71 @@ - (CFIndex)getIndexForIdentifier:(ABMultiValueIdentifier)identifier {
     return index;
 }
 
+- (_ABMultiValue*)createMutableCopy {
+    _ABMultiValue* copy = [[_ABMultiValue alloc] initWithPropertyType:[self getPropertyType]];
+    for (__ABMultiValuePair* pair in self->_list) {
+        [copy appendPairWithLabel:pair.label andValue:pair.value];
+    }
+
+    return copy;
+}
+
+- (bool)addValue:(CFTypeRef)value andLabel:(CFStringRef)label outIdentifier:(ABMultiValueIdentifier*)outIdentifier {
+    bool result = [self appendPairWithLabel:(__bridge NSString*)label andValue:(__bridge id)value];
+    if (result && outIdentifier) {
+        *outIdentifier = [self getIdentifierAtIndex:([self getCount] - 1)];
```

Comment:
Yes, because we only enter this case if appendPairWithLabel succeeded, which increments the count.

---
In reply to: [75513394](https://github.com/Microsoft/WinObjC/pull/893#discussion_r75513394) [](ancestors = 75513394)